releasenotes/notes/2.0/prepare-2.0.0-bab067ae93d40bb1.yaml

```diff
@@ -0,0 +1,35 @@
+---
+prelude: >
+    We are pleased to release Qiskit v2.0.0, with new features that improve performance and its capabilities.
+    The feature highlights of Qiskit v2.0.0 include:
+
+      * The introduction of a C API for building and interacting with
+        :class:`.SparseObservable` objects. This first 
+        compiled language interface is an experimental feature that will 
```

Comment:
We should probably be explicit here and say that an experimental feature means we reserve the right to change the API in a minor version per https://docs.quantum.ibm.com/open-source/qiskit-sdk-version-strategy